client/src/components/common/applicant-info-nav.vue

```diff
@@ -81,13 +81,14 @@ export default class ApplicantInfoNav extends Vue {
     }
     newReqModule.mutateSubmissionTabNumber(this.tab + 1)
   }
+
   async submit () {
     const { nrId } = this
     if (this.editMode) {
       await newReqModule.patchNameRequests()
       await newReqModule.checkinNameRequest()
       timerModule.stopTimer({ id: this.$EXISTING_NR_TIMER_NAME })
-      this.fetchNr(nrId).then(() => {})
+      await this.fetchNr(+nrId)
```

Comment:
In some places, `nrId` is a number (which is correct), but in other places it's a string or any.

This ticket is complicated enough as-is, but at some point we should fix the typings.